egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
>You can assume that durmodel_train.sh is always going to be called with 3 arguments. In case the user wants to use dur_egs prepared else where you can support this using the --egs-dir option. Take a look at steps/nnet3/lstm/train.sh to see an example of this.

I did not follow you exactly. For simplicity I have put everything (such as the model, examples, raw nnets, logs, ...) in a <duration-model-dir> (for eg. exp/tri3/durmodel) and the first argument to train.sh is responsible for this. Would you mean that I add the option for user to put the model in <duration-model-dir> and the examples somewhere else ?
Nevertheless, I could require the user to always provide 3 arguments. I just added this 1-or-3-arguments option to make the call simpler when we only want to train (stage>0).